ibm_db2/datadog_checks/ibm_db2/ibm_db2.py

```diff
@@ -65,6 +75,21 @@ def check(self, instance):
         self.query_table_space()
         self.query_transaction_log()
         self.query_custom()
+        self.collect_metadata()
+
+    def collect_metadata(self):
+        try:
+            version = get_version(self._conn)
+        except Exception as e:
+            self.log.error("Error getting version: {}".format(str(e)))
+            return
+
+        if version is not None:
+            self.set_metadata('version', version, scheme='regex', pattern=self.VERSION_REGEX)
```

Comment:
The scheme will be sent as `version.scheme = regex`, I don't think that provide a good value. Maybe `scheme=custom` would be better? Or something more specific? `scheme=ibm_version_format`

Do you have any idea if this custom format is specific to ibm_db2 or if it's shared with other services?

Reply:
EDIT: scheme can probably be useful to know how to compare versions. `custom` is too wide let's not use that

Reply:
https://github.com/DataDog/integrations-core/blob/80b3ff4c17cec66f6538bbf928b4ed2b6ef931bb/datadog_checks_base/datadog_checks/base/utils/metadata/core.py#L66-L68